src/importchecker/tests/fixture/absimport.py

```diff
@@ -1,2 +1,2 @@
-import sys
-import sys.stderr
+import sys  # noqa: F401 imported but unused
+import sys.stderr  # noqa: F401 imported but unused
```

Comment:
Uh, could we tell flake8 not to check anything under `tests/fixture/`?

(I'm guessing not easily with the current metaconfiguration system we have.)